buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/IntegrationTest.kt

```diff
@@ -64,4 +65,17 @@ open class IntegrationTest : DistributionTest() {
             "embedded" -> TimeUnit.MINUTES.toMillis(30)
             else -> TimeUnit.MINUTES.toMillis(165) // 2h45m
         }
+
+    override fun setClasspath(classpath: FileCollection) {
+        /*
+         * The 'kotlin-daemon-client.jar' repackages 'native-platform' with all its binaries.
+         * Here we Make sure it is paced at the end of the test classpath so that we do not accidentally
```

Comment:
```suggestion
         * Here we make sure it is placed at the end of the test classpath so that we do not accidentally
```

Reply:
Thanks @eskatos. Fixed in cf86ea14e63b506bddea7776184de15e94bfa0b6.